### ISSUE TITLE
With certain documents, preview panel doesn't open scrolled to the top

### ISSUE THREAD

**Issue by UrsineRaven** [mohzy83/NppMarkdownPanel#17]
Steps to reproduce:
1. Open Notepad++ with a markdown file that will cause the preview to need scrolling
2. Click the preview toolbar button
3. The preview isn't at the top of the file (you have to scroll up to see the beginning)

**Comment by UrsineRaven** [comment]
After further testing, this doesn't appear to happen to every document, and it's unrelated to the docked state.

It happens consistently to a markdown file I have that starts with:
```markdown
# Notes from training

----

## Definitions

### What is [REDACTED]?
```

**Comment by mohzy83** [comment]
This bug is now fixed in version 0.7.0
